includes/footer.php

```diff
@@ -1,21 +1,91 @@
+                    </ol>
+
+                    <!-- Slides -->
+                    <div class="carousel-inner" role="listbox">
+                         <div class="item active">
+                            <img src="/img/sponsors/ehs.jpg" alt="Eastlake High School Logo" title="Eastlake High School"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/google.jpg" alt="Google Logo" title="Google"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/lwsd.jpg" alt="Lake Washington School Foundation Logo" title="Lake Washington School Foundation"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/boeing.jpg" alt="Boeing Logo" title="Boeing"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/chase.jpg" alt="Chase Logo" title="Chase"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/FIRSTWa.jpg" alt="FIRST Washington Logo" title="FIRST Washington"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/godaddy.jpg" alt="GoDaddy Logo" title="GoDaddy"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/honeywell.jpg" alt="Honeywell Logo" title="Honeywell"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/kenworth.jpg" alt="Kenworth Logo" title="Kenworth"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/microsoft.jpg" alt="Microsoft Logo" title="Microsoft"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/ospi.jpg" alt="Office of the Superintendant of Public health Logo" title="Office of the Superintendent of Public Instruction"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/rocketdyne.jpg" alt="Aerojet Rocketdyne Logo" title="Aerojet Rocketdyne"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/sae.jpg" alt="SAE Washington Logo" title="SAE Washington"/>
+                        </div>
                     </div>
-                    <hr>
+
+                    <!-- Left and right controls -->
+                    <style type="text/css">
+                        .left, .right {
+                            background: none !important;
```

Comment:
So, it looks like you removed the background gradient (:+1:), but now it makes the arrows practically invisible. Can you change them to a black/grey?